domains/Accounts/Middleware/ThrottleGuestMiddleware.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Domains\Accounts\Middleware;
+
+use Closure;
+use GrahamCampbell\Throttle\Http\Middleware\ThrottleMiddleware as ThrottleMiddleware;
+use GrahamCampbell\Throttle\Throttle;
+use Illuminate\Contracts\Auth\Factory as Auth;
+
+class ThrottleGuestMiddleware extends ThrottleMiddleware
```

Comment:
Do we need this middleware? Can't we just use the base one and, in the route, do:
`'middleware' => 'throtte:30,1',`